### ISSUE TITLE
Compatibility with Python 3.6

### ISSUE THREAD

**Issue by dariemihai95** [ag-ds-bubble/swtloc#1]
Best library of this kind, but had a hell of a time setting it up in a virtual environment.

**Why ?**

- Ubuntu 16 and 18 comes by default with Python 3.6 installed.
- Docker tensorflow/tensorflow images come with Python 3.6.

**Workaround ?**

- Unfortunately, no. I tried ignoring Python version error at installation but resulted in the library stopping altogether. It crashed when called time.perf_counter_ns() as this is present from python 3.7 and above.

- Tried using a custom built docker image* and even building one from scratch but resulted in other massive overhead on my end.

**Contribution**

At the moment I am unable to highlight exactly what needs to be changed besides what I mentioned earlier but if needed I can help find a solution.

**Benefits**

Greater compatibility with older and more stable versions of python and also with other environments such as the one I'm trying to use.

**Comment by ag-ds-bubble** [comment]
Hey @dariemihai95, i have tested this library in MacOS and Windows on Python3.7 & Python3.8, although yet to try out for the older versions of Python as you mentioned. Will be fixed by the next release most probably. Can you please elaborate on what did you mean in your second comment in the **Workaround**, regarding `Tried using a custom built tensorflow...`

**Comment by dariemihai95** [comment]
@ag-ds-bubble had a mistake there, what I meant was a custom docker image. Corrected it. Thank you for pointing out.
Tested it on Ubuntu 20.04 with python 3.8 and in docker with an image based on ubuntu 18 with python 3.6 as mentioned above. If you need any pr tested on those environments I can help out.

**Comment by ag-ds-bubble** [comment]
Hello @dariemihai95, Support for Python 3.6 is now live. Check it out!
I have tested v1.0.0.3 on Windows and Mac on Pip and Conda venv, although unsure about Ubuntu. Cheers.

Although i am awaiting your feedback for the Ubuntu setup, whether you are able to setup the package with ease. Will close this issue then.

**Comment by dariemihai95** [comment]
I have successfully tested it in Ubuntu 18 with python 3.6 and it works without issues. I have yet to test it in a docker container but I don't see why it would have any issues. One thing I noticed though but I may be wrong is that it takes a while longer to process files with this new version. I will test this out and if I have some conclusive results I will let you know. Thank you and great job so far !
